### PR TITLE
Make top progress bar more visible

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/10383

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-pkg/src/components/LoadingIndicator.vue [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <template>
-  <div v-if="isLoading" id="oc-loading-indicator">
+  <div v-if="true" id="oc-loading-indicator">
```
Comment (JanAckermann) [inline packages/web-pkg/src/components/LoadingIndicator.vue:2]: For easy testing

Comment (JanAckermann) [inline packages/web-pkg/src/components/LoadingIndicator.vue:2]:
```suggestion
  <div v-if="isLoading" id="oc-loading-indicator">
```

File: packages/web-pkg/src/components/LoadingIndicator.vue [hunk 2]
```diff
@@ -4,7 +4,7 @@
       :max="100"
       :indeterminate="currentProgress === null"
       :value="currentProgress"
-      size="small"
+      size="medium"
```
Comment (JanAckermann) [inline packages/web-pkg/src/components/LoadingIndicator.vue:7]:
```suggestion
      size="small"
```

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**